src/escapei.rs

```diff
@@ -112,7 +112,14 @@ fn _escape<F: Fn(u8) -> bool>(raw: &str, escape_chars: F) -> Cow<str> {
             b'\'' => escaped.extend_from_slice(b"&apos;"),
             b'&' => escaped.extend_from_slice(b"&amp;"),
             b'"' => escaped.extend_from_slice(b"&quot;"),
-            _ => unreachable!("Only '<', '>','\', '&' and '\"' are escaped"),
+
+            b'\t' => escaped.extend_from_slice(b"&#9;"),
```

Comment:
Leave a comment about the reasoning behind why these characters (sometimes) need to be escaped as it isn't immediately obvious

Reply:
Commend added